src/coreclr/scripts/superpmi.py

```diff
@@ -600,7 +596,18 @@ def run_to_completion(self, async_callback, *extra_args):
         """
 
         reset_env = os.environ.copy()
-        loop = asyncio.get_event_loop()
+
+        try:
+            loop = asyncio.get_running_loop()
+        except RuntimeError:
+            if 'win32' in sys.platform:
+                # Windows specific event-loop policy & cmd
+                loop = asyncio.ProactorEventLoop()
```

Comment:
Seems odd that there is something win32-specific. Why is that?

Reply:
This is a good question, I'm not super familiar with it - but I didn't want to change the existing behavior.

Reply:
https://docs.python.org/3/library/asyncio-platforms.html seems to indicate that ProactorEventLoop is now the default on Windows, making this unnecessary.

Did you find some documentation suggesting that this `get`/`new`/`set` pattern is the standard practice now?  https://docs.python.org/3/library/asyncio-eventloop.html says that these are low-level methods that most applications won't need to use.  https://docs.python.org/3/library/asyncio-runner.html and some other places suggest that a standard way to go is to call `asyncio.run(main())` on an `async def main`.  Then an event loop is in place, though I suspect this then suggests using `asyncio.create_task`.  Unfortunately, I don't know more than what I've just read about to write this paragraph.

(side note, especially if you're looking at version requirements - the check in `main` appears to be incorrect, needing an `or` instead of an `and`, though using `sys.version_info` and tuple comparison would be a bit better.)